runelite-api/src/main/java/net/runelite/api/widgets/WidgetID.java

```diff
@@ -918,4 +919,9 @@
 	{
 		static final int CONTAINER = 2;
 	}
+
+	static class BankPin
+	{
+		static final int CONTAINER = 1;
```

Comment:
`213.1` seems to be the background where `213.0` is the container (since it contains the buttons, text, border elements, etc.)